### PR TITLE
added in quit to stop server, added logging to server and file load

### PR DESCRIPTION
**Pull request by erictu**
As a result, the `-print_metrics` option in `mango-submit` now works as well.

### CODE REVIEW

File: mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala [hunk 1]
```diff
@@ -102,6 +103,27 @@ object VizReads extends ADAMCommandCompanion {
     }
     freqBuffer.toList
   }
+
+  def quit() {
+    val thread = new Thread {
```
Comment (fnothaft) [inline mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala]: Why do you need to do this as a thread?

Comment (erictu) [inline mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala]: If you don't do it in a thread, the server will stop (throws an error about sleep being interrupted), but the VizReads process will keep running and won't hand back the terminal.

Comment (fnothaft) [inline mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala]: Ah, that makes sense! Thanks!

**Comment by fnothaft** [comment]
Merged! Thanks @erictu!

**Pull request merged**